db/controllers/comments-controller.js

```diff
@@ -7,4 +8,14 @@ exports.getComments = (req, res, next) => {
         res.status(200).send({comments})
         })
     .catch(next)
+}
+
+exports.postComments = (req, res, next) => {
+    const { article_id } = req.params;
+    const { username, body } = req.body;
+    return insertIntoComments(article_id,username,body)
+        .then((comment) => {
+        res.status(201).send({comment})
+        })
```

Comment:
^ lovely controller! Although I might edit the name to be 'postComment', as there's only one comment posted at a time!